src/mainwindow.cpp

```diff
@@ -2944,7 +2944,7 @@ void MainWindow::notesWereModified(const QString &str) {
 
         // rebuild and reload the notes directory list
         buildNotesIndexAndLoadNoteDirectoryList();
-        setCurrentNote(std::move(this->currentNote), false);
+        setCurrentNote(std::move(this->currentNote), true);
```

Comment:
I don't think that's a good idea... If any random note in the note folder was changed it would trigger that the note text edit is refreshed and that is very bad while you are editing the note.